composer.json

```diff
@@ -19,7 +19,8 @@
     ],
     "require": {
         "php": ">=7.4.0",
-        "ext-uopz": ">=6.1.1"
+        "ext-uopz": ">=6.1.1",
+        "ext-calendar": "*"
```

Comment:
would it make sense to not require this and mock unixtojd only if function exists, or extension is loaded? This way we don't force everyone using this library to have this extension (even though it seems enabled by default in most use cases).

Reply:
@asprega yes, that sounds like a good idea. I've updated the PR